presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -149,6 +151,15 @@ public SystemSessionProperties(
                         false,
                         value -> JoinDistributionType.valueOf(((String) value).toUpperCase()),
                         JoinDistributionType::name),
+                new PropertyMetadata<>(
+                        JOIN_MAX_BROADCAST_TABLE_SIZE,
+                        "Maximum size of a table that can be broadcast for JOIN (0 disables the limit). The most pessimistic size estimate is taken when doing comparison.",
```

Comment:
`The most pessimistic size estimate is taken when doing comparison.` - This doesn't seem to be the case, is it?

Reply:
> 0 disables the limit

It should be rather `+INF` to disable limit

Reply:
> The most pessimistic size estimate is taken when doing comparison.

I would prefer to not to go to such details.

> Maximum size

Please update to `Maximum estimated size`